.gitignore

```diff
@@ -9,3 +9,5 @@ pom.xml.asc
 .nrepl-port
 reports
 .cpcache
+.clj-kondo
+.lsp
```

Comment:
Changes that aren't relevant to the PR should ideally be omitted.

Reply:
These are both standard things that clojure-lsp/IDE's might pull in, so figured they might be useful to add. I can yank it though if you really don't want them included

Reply:
They are useful, but they're just not relevant to the current PR. A PR should *only* contain code to implement the feature/fix described.

Reply:
Ok I can yoink them then if them's the rules.